lib/js-api/src/api/gs-core-rest-api/api/legal-hold.api.ts

```diff
@@ -61,35 +60,34 @@ export class LegalHoldApi extends BaseApi {
      * @param nodeIds one element list with id of nodes to assign to existing hold
      * @returns Promise<HoldEntry>
      */
-    assignHold(nodeIds: {id: string}[], holdId: string): Promise<HoldEntry> {
+    assignHold(nodeIds: { id: string }[], holdId: string): Promise<HoldEntry> {
```

Comment:
If docs for that method are already added they should be adjusted as well

Reply:
should be done in scope of https://github.com/Alfresco/alfresco-ng2-components/pull/9841
@tomson7777